tests/auth0-session/fixtures/server.ts

```diff
@@ -94,9 +93,10 @@ const createHandlers = (params: ConfigParameters): Handlers => {
   };
 };
 
-const jsonParse = bodyParser.json();
-const parseJson = (req: IncomingMessage, res: ServerResponse): Promise<IncomingMessage> =>
-  new Promise((resolve, reject) => {
+export const parseJson = async (req: IncomingMessage, res: ServerResponse): Promise<IncomingMessage> => {
+  const { default: bodyParser } = await import('body-parser');
```

Comment:
Just curious, what is the reason we need a dynamic import here?

Reply:
I added  a dependency on this file in `tests/fixtures/server.ts` which some test files that run edge tests depend on and `body-parser` can't be imported on the edge runtime (it must use `eval` or something)